v2.0/enable-node-map.md

```diff
@@ -1,7 +1,200 @@
 ---
-title: Enable the Node Map
+title: Enable the Node Map (Enterprise)
```

Comment:
Just realized that we don't put `(Enterprise)` in the title for `BACKUP`/`RESTORE`. Let's also remove that from this page as well as the Table Partitioning page. However, please leave it in the sidenav and in the callouts.

Reply:
Removed from this page. Made a note to remove from the partitioning doc.